src/components/AdminOrderItem/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { Avatar, Checkbox, Form, Input, Modal, Row, Col } from 'antd';
+import Icon from '@ant-design/icons';
+import { ReactComponent as CommentBoxEmpty } from '../../assets/icons/CommentBoxEmpty.svg';
+
+import './style.less';
+
+interface AdminOrderItemProps {
+  orderItem: {
+    uuid: string;
+    itemName: string;
+    quantity: number;
+    fulfilled: boolean;
+    price: number;
+    description: string;
+    notes: string;
+  };
+}
+
+const AdminOrderItem: React.FC<AdminOrderItemProps> = (props) => {
+  const [visible, setVisible] = useState(false);
+  const [note, setNote] = useState('');
+  const [fulfilled, setFulfilled] = useState(false);
+  const showNote = () => {
+    setVisible(true);
+  };
+
+  const closeNote = () => {
+    setVisible(false);
+  };
+
+  const handleOk = () => {
+    // For now, console statements. These will be removed
+    // when the API is implemented in actions.
+    // eslint-disable-next-line no-console
+    console.log(`Update note is called here! Contents: ${note}`);
+    setVisible(false);
+  };
+
+  const { orderItem } = props;
+  return (
```

Comment:
Hmm why put the hooks and functionality here versus pulling it up to the container then passing it down as props?

Reply:
Yeah, I did indeed leave functionality and props within the components and not the container, my bad. While fixing this, though, I've noticed a bit of scope creep in this PR.

This PR was supposed to implement the design of the components, not their middleware; in the process of moving this functionality up to the container, I realized I needed to write up actions and reducers for the collection, something already partially written in #315. As such, it's likely merge conflicts will happen next merge window. We probably to setup the Redux store before we continue writing more middleware.

For the purposes of keeping this PR clean, I will remove the functionality from the AdminOrderItem component (except for some `console.log` calls to test whether specific events do trigger) and move `testProps` up to the container.

Reply:
Moved up the functionality in commit  8c65bd8.